cnf-certification-test/accesscontrol/suite.go

```diff
@@ -474,3 +479,23 @@ func TestOneProcessPerContainer(env *provider.TestEnvironment) {
 		ginkgo.Fail(errMsg)
 	}
 }
+
+func TestSYSNiceRealtimeCapability(env *provider.TestEnvironment) {
+	var podsWithoutSysNice []string
+
+	// Loop through all of the labeled containers and compare their security context capabilities and whether
+	// or not the node's kernel is realtime enabled.
+	for _, cut := range env.Containers {
+		n := env.Nodes[cut.NodeName]
+		if n.IsRTKernel() && !strings.Contains(cut.Data.SecurityContext.Capabilities.String(), "SYS_NICE") {
+			logrus.Debugf("Container: %s has been found running on a realtime kernel enabled node without SYS_NICE capability.", cut.String())
+			podsWithoutSysNice = append(podsWithoutSysNice, cut.String())
+		}
+	}
+
+	if n := len(podsWithoutSysNice); n > 0 {
+		errMsg := fmt.Sprintf("Number of containers running on realtime kernels without SYS_NICE: %d", n)
+		tnf.ClaimFilePrintf(errMsg)
```

Comment:
ClaimFilePrintf can be used like fmt.Printf/logrus.Infof, so you don't need the previous fmt.Sprintf:
```
tnf.ClaimFilePrintf("Number of containers running on realtime kernels without SYS_NICE: %d", n)
```

Reply:
I just didn't want to duplicate the string.

Reply:
My fault, sorry...